vscode-extension/README.md

```diff
@@ -16,7 +16,7 @@ Search for "Relay for VSCode" in the VS Code extensions panel or install through
   - We support standard config file formats (`.yml`, `.js`, `.json`), and the the `relay` field in your `package.json`
```

Comment:
Can we add a "Credits" section at the bottom of this file that acknowledges where we got these files? That will help future contributors know where they came from and also make it clear that this is not our original work.

Also, let's drop this PR in the `#graphql-lsp` discord channel to let them know our plans. Since we're taking their code, I'd like to make sure they feel good about it and don't get the impression that we are trying to pass their code off as our own.

Reply:
https://discord.com/channels/625400653321076807/863688312756371476/976322813650665573

Reply:
is not good...

Reply:
is there a way to define extension dependencies somehow? Saying that in order for relay extension to work the `graphql` extension is also have to be installed? Or maybe create an extension pack?

Reply:
or can we install this grammar via `npm` somehow?

Reply:
I mean we could definitely pull it off through an npm dep. Requiring it as an extension would cause conflicts with the Relay extension potentially. We strictly want the grammars.

Can we elaborate on that "is not good..." comment?